src/texstudio.cpp

```diff
@@ -10959,4 +10969,9 @@ void Texstudio::openBugsAndFeatures() {
 	QDesktopServices::openUrl(QUrl("https://github.com/texstudio-org/texstudio/issues/"));
 }
 
+void Texstudio::tocItemDoubleClicked(QString section, QString file, int line)
```

Comment:
txs uses single click for jumping, it makes no sense to have different use mechanisms